<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suitenameshoot" duration-ms="12981" started-at="2019-09-26T17:33:29Z" finished-at="2019-09-26T17:33:42Z">
    <groups>
      <group name="smoke">
        <method signature="testnggcls.Functionalityies()[pri:0, instance:testnggpackgee.testnggcls@58651fd0]" name="Functionalityies" class="testnggpackgee.testnggcls"/>
        <method signature="testnggcls.Login()[pri:0, instance:testnggpackgee.testnggcls@58651fd0]" name="Login" class="testnggpackgee.testnggcls"/>
      </group> <!-- smoke -->
    </groups>
    <test name="testnamevj" duration-ms="12981" started-at="2019-09-26T17:33:29Z" finished-at="2019-09-26T17:33:42Z">
      <class name="testnggpackgee.testnggcls">
        <test-method status="FAIL" signature="Functionalityies()[pri:0, instance:testnggpackgee.testnggcls@58651fd0]" name="Functionalityies" duration-ms="679" started-at="2019-09-26T23:03:29Z" finished-at="2019-09-26T23:03:30Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at excellpackage.excellclssie.inputss(excellclssie.java:17)
	at testnggpackgee.testnggcls.Functionalityies(testnggcls.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Functionalityies -->
        <test-method status="PASS" signature="Login()[pri:0, instance:testnggpackgee.testnggcls@58651fd0]" name="Login" duration-ms="12267" started-at="2019-09-26T23:03:30Z" finished-at="2019-09-26T23:03:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
      </class> <!-- testnggpackgee.testnggcls -->
    </test> <!-- testnamevj -->
  </suite> <!-- Suitenameshoot -->
</testng-results>
